src/outputs/parthenon_hdf5.hpp

```diff
@@ -62,11 +119,11 @@ using parthenon::Real;
         }                                                                                \
       }                                                                                  \
     }                                                                                    \
-  }
+  } while (false)
```

Comment:
wait, aren't these in use? Why are you disabling the loop?

Reply:
This doesn't disable the loop - it's just good macro hygiene. See: https://stackoverflow.com/a/4674580

Reply:
Ah, I see. Didn't know about this. Thanks.